akka-docs/src/main/paradox/cluster-sharding.md

```diff
@@ -5,7 +5,7 @@
 Akka Classic is the original Actor APIs, which have been improved by more type safe and guided Actor APIs, 
 known as Akka Typed. Akka Classic is still fully supported and existing applications can continue to use 
 the classic APIs. It is also possible to use Akka Typed together with classic actors within the same 
-ActorSystem, see @ref[coexistense](typed/coexisting.md). For new projects we recommend using the new Actor APIs.
+ActorSystem, see @ref[coexistence](typed/coexisting.md). For new projects we recommend using the new Actor APIs.
```

Comment:
search for it, there are plenty of the same that should also be changed

Reply:
Thanks. Pushed.